libraries/standard/mqtt/src/mqtt.c

```diff
@@ -264,6 +305,13 @@ static uint32_t calculateElapsedTime( uint32_t later,
 
 /*-----------------------------------------------------------*/
 
+static uint32_t getTimeStampNoSystemTime()
```

Comment:
Can this dummy function be avoided? It seems like a technical debt introduced for supporting existing private functions that require the `getTime` function...

Reply:
Latest commit in #990 removes this